sdk/cognitivelanguage/ai-language-text/test/public/utils/customTestHelpter.ts

```diff
@@ -167,5 +164,5 @@ export async function cleanupCustomTestResource(
   const deleteTask = await client
     .path("/authoring/analyze-text/projects/{projectName}", projectName)
     .delete();
-  await polling(client, deleteTask);
+  polling(client, deleteTask);
```

Comment:
Do we need to poll for it? I think we don't need to.

Reply:
We actually don't. I'll remove it